backend/accessibility_templates/aa-004.xml

```diff
@@ -0,0 +1,21 @@
+<AccessibilityAttributes>
+ <SubtitleAttributes>
+  <Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023::3"/>
+  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:4"/>
+  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:99"/>
+  <SubtitleLanguage>zh-hans</SubtitleLanguage>
+  <SuitableForTTS>false</SuitableForTTS>
+ </SubtitleAttributes>
+ <AudioDescriptionAttributes>
+  <AudioAttributes>
+   <AudioLanguage>tgr</AudioLanguage>
+   <BitRate>3000</BitRate>
```

Comment:
Bit Rate can possibly omitted.

Reply:
Indeed, `<Bitrate>` is not included in the A177 profile of `<AudioAttributes>` - see clause 6.10.10 of A177r6.
Will be removed from the accessibility templates.